ldap/servers/plugins/uiduniq/uid.c

```diff
@@ -1392,13 +1392,13 @@ preop_modrdn(Slapi_PBlock *pb)
              */
             if (NULL != markerObjectClass) {
                 /* Subtree defined by location of marker object class */
-                result = findSubtreeAndSearch(slapi_entry_get_sdn(e), attrNames, attr, NULL,
-                                              requiredObjectClass, superior,
+                result = findSubtreeAndSearch(superiorSDN, attrNames, attr, NULL,
+                                              requiredObjectClass, sourceSDN,
```

Comment:
Instead of sourceSDN, shouldn't it be <rdn>,<superiorSDN> ?

Reply:
Why? We need sourceSDN so we can ignore it in the results.

Reply:
Your change looks valid to me, using destinationSDN and sourceSDN